common/envHandler.js

```diff
@@ -34,8 +35,60 @@ envHandler.get = function (envName, cb) {
 };
 
 envHandler.put = function (envName, envValue, cb) {
+  var bag = {
+    envName: envName,
+    envValue: envValue
+  };
+
+  async.series(
+    [
+      _createEnvVar.bind(null, bag),
+      _updateEnvVar.bind(null, bag)
+    ],
+    function (err) {
+      if (err)
+        return cb(err, null);
+
+      return cb(null);
+    }
+  );
+};
+
+// adds env variable ENV_VAR="" to admiral.env file if it doesn't exist
+function _createEnvVar(bag, next) {
+  var createCmd = util.format('sudo grep -qo \'%s\' %s || echo \'%s=""\' >> %s '
```

Comment:
- we cant do this. you're changing put to basically do an upsert. so use the get method to find the env, if its not there, then add a function to insert it else update it. we cant do everything in one bash script. makes things unreadable and clunky

Reply:
@ric03uec : ok, i 'll create a new route to `post admiral env` and move the logic there